### PR TITLE
Hub - unify on "Create execution environment", fix test failing because of it

### PR DESCRIPTION
**Pull request by himdel**
The `can add and delete a new execution environment` test in `cypress/e2e/hub/execution-environments.cy.ts` fails sporadically, because when there are no EEs, the button has `data-cy=create-execution-environment`, while when there are EEs, the button has `data-cy=add-execution-environment`. These labels are generated from the button text, which is also inconsistent.

Fixing the button text to say "Create execution environment" on both the empty state screen and the list screen.

(the only place remaining that says "Add execution environment" is now `useAwxRoleMetadata`, which seems related to awx role names)

### CODE REVIEW

**Comment by himdel** [comment]
Cc @tiyiprh is "Create execution environment" for both buttons the right direction?  :)

**Review by ZitaNemeckova — APPROVED** [review]
Code-wise 👍 Waiting on UXD approval.

**Comment by ZitaNemeckova** [comment]
@himdel Let's merge this. There's no consistency across projects yet.

**Comment by himdel** [comment]
Agreed, we can always change it later, one less skipped test :)

**Comment by tiyiprh** [comment]
> Cc @tiyiprh is "Create execution environment" for both buttons the right direction? :)

Sorry @himdel was on PTO last week but yes that's correct!

**Comment by himdel** [comment]
Thanks!, no worries, sorry for rushing you :)

**Pull request merged**
